api/src/pcapi/core/offerers/synchronize_venues_banners_with_google_places.py

```diff
@@ -181,6 +185,11 @@ def synchronize_venues_banners_with_google_places(
             "nb_places_with_owner_photo": nb_places_with_owner_photo,
         },
     )
+    if images_ignored_due_to_ratio:
+        logger.warning(
+            "Image ignored due to ratio: %s",
+            images_ignored_due_to_ratio,
+        )
```

Comment:
[question] Est-ce qu'on est simplement intéressé par le **nombre** de lieux problématiques ? Ne voudrait-on pas logger les identifiants de ces lieux (et, si besoin, leur nombre, en plus) ?

Reply:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**🎯 suggestion:** ‏
Un log de la forme ` "Image ignored due to ratio: %s ; first 10 ids: %s", number, ids[:10]` pour avoir un échantillon pour investiguer

Reply:
je vais plutôt mettre tout ces venue en extra, comme ça si on résoud le problème on peux relancer que pour ces venues.